src/components/InstanceListView/InstanceTable.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+
+import { IClusterServiceClass } from "../../shared/ClusterServiceClass";
+import { IServiceInstance } from "../../shared/ServiceInstance";
+import { Card, CardContainer } from "../Card";
+
+export interface InstanceTableProps {
+  classes: IClusterServiceClass[];
+  instances: IServiceInstance[];
+}
+
+export const InstanceTable = (props: InstanceTableProps) => {
+  const { instances, classes } = props;
+  return (
+    <div className="InstanceTable">
+      <table>
+        <tbody>
```

Comment:
A `div` element inside `tbody` is not valid and React shows errors in the console. Change the `table > tbody` container for a `section`.